apps/trading-e2e/src/integration/trading-orders.cy.ts

```diff
@@ -221,7 +221,10 @@ describe('subscribe orders', { tags: '@smoke' }, () => {
       id: orderId,
       status: Schema.OrderStatus.STATUS_PARKED,
     });
-    cy.getByTestId(`order-status-${orderId}`).should('have.text', 'Parked');
+    cy.getByTestId(`order-status-${orderId}`).should(
+      'have.text',
+      'Parked: Internal error'
```

Comment:
@radoslaw-szpiech here I have a comment that I am n ot sure if I should keep anymore.
```
    // 7002-SORD-048
    // NOT COVERED:   must see an explanation of why parked orders happen
```

Reply:
we are now showing the rejection reason but the rejection reason in this test is "Internal error". It could be sth else, do we need to change the mock? should we?

Reply:
I think it's cool. Please, remove `NOT COVERED` comment, leave `SORD` for AC coverage.

Reply:
awesome. updated.